### PR TITLE
Fix openshift explorer not realizing it is logged out of a cluster

### LINKED ISSUE #506: OpenShift Explorer sometimes does not detect when current session is expired and new loging required
This happens because detection is done based on stdout content and extension does not have all possible messages that indicates expired session.

### PR DESCRIPTION
**Pull request by jrichter1**
Because it could not find the output that would say login is required.
Fix #506.

### CODE REVIEW

File: src/odo.ts [hunk 1]
```diff
@@ -298,7 +298,7 @@ export class OdoImpl implements Odo {
         const result: cliInstance.CliExitData = await this.execute(
             Command.printOdoVersionAndProjects(), process.cwd(), false
         );
-        if (result.stdout.indexOf('Please log in to the cluster') > -1 || result.stdout.indexOf('the server has asked for the client to provide credentials') > -1) {
+        if (result.stdout.indexOf('Please login to your server') > -1 || result.stdout.indexOf('the server has asked for the client to provide credentials') > -1) {
```
Comment (mohitsuman) [inline src/odo.ts]: The odo(0.0.16) output shows the following message during login: `Please log in to the cluster`.

Comment (jrichter1) [reply src/odo.ts]: not for me, it doesn't

Comment (jrichter1) [reply src/odo.ts]: right, there are in fact two spots in odo code that fail on login with different messages

Please log in:
https://github.com/redhat-developer/odo/blob/eec9a26ec4b7520e90a1d94370c3ce92f84d5f59/pkg/occlient/occlient.go#L246

Please login:
https://github.com/redhat-developer/odo/blob/eec9a26ec4b7520e90a1d94370c3ce92f84d5f59/pkg/occlient/occlient.go#L182 

No idea what the differentiating factor is.

Comment (jrichter1) [reply src/odo.ts]: Hah!
```

Comment (jeffmaury) [reply src/odo.ts]: We should open an issue on the odo side, WDYT ?

Comment (mohitsuman) [reply src/odo.ts]: We need to open an issue on odo side then.

Comment (jrichter1) [reply src/odo.ts]: I don't think this is odo's fault, it makes sense to have different errors for these two cases. We just haven't realized this path exists in odo.

Comment (dgolovin) [reply src/odo.ts]: It might be just message from openshift and it could be different for different versions.
But anyway it worth to open issue for odo.

**Pull request merged**